src/CoreShop2VueStorefrontBundle/Bridge/Response/ResponseBodyCreator.php

```diff
@@ -99,7 +99,7 @@ private function getAddress(AddressInterface $defaultAddress, int $customerId):
         $address['region']['region'] = null;
         $address['region']['region_id'] = 0;
         $address['region_id'] = 0;
-        $address['country_id'] = $defaultAddress->getCountry();
+        $address['country_id'] = $defaultAddress->getCountry()->getIsoCode();
```

Comment:
@danteYuX I think that we've to check if getCountry return value to avoid problem with NullObject

Reply:
In this case I think it's the best to check before that the object is null or not.